packages/documentation/src/pages/getting-started/common-tasks/releases.mdx

```diff
@@ -1,109 +0,0 @@
-
-Because `master` is designed to always be deployable, long running features that should _not_ be deployed should utilize feature flags in the code that disable the feature for the actual production environment. Notifying the DevOps team on what feature flags should be enabled/disabled in staging and production environment is an important part of this process. However, it's likely that breakages in staging will occur and that this is necessary to discover these prior to moving anything to the production steps.
-
-Speaking of production steps, things kick off with the daily production push
-
-> 6\. Create a release in GitHub from master, tag artifacts of that commit sha with release name
-
-Every work day at the configured time a [Jenkins automerge job](http://jenkins.vetsgov-internal/job/deploys/job/vets-gov-automerge/) sends a link to the #vetsgov-engineers Slack channel with a diff between the last release and the most recent changes in `master`. This commit reference is stored to ensure the diff and released version is deterministic.
-
-After a time has elapsed ( currently set to 60m ) release is created at the reference from above.
-
-> 7\. Automated: Deploy to production using artifacts
-
-From here, Jenkins can kickoff a production deployment. After the deployment occurs, the normal site monitoring infrastructure will be used to valide it is working. As this process is automatic any new features should have monitoring in place before, or as a part of their deployment
-
-## Other Notes
-
-- The code that appears in the `master` branch _actually_ gets deployed to both dev and staging environments. This is done to support different configurations for the DevOps team as they work to support any configuration changes (i.e. in dev first).
-
-## Deployment Rollbacks
-
-If a production deployment introduces issues that affect Service Level Objectives (SLOs) established for the project, the DevOps team may restore service to users by rolling back the deployment. This is accomplished by triggering a new deploy job in Jenkins using a previous release tag.
-
-The use of hotfixes is discouraged, but may be useful in an emergency situation when `master` has significantly deviated from the release and a fix to the failed production release is critical. To create a hotfix, create a branch from the last stable release tag, make changes necessary (with review), create a new release tag following the correct naming scheme, and trigger a deploy in Jenkins with the release name as a parameter.
-
-If SLOs are not affected and a fix is not critical, no rollback will be issued. Instead the fix should be applied through the standard development workflow.
-
-# Getting a change from Drupal or vagov-content into production
-
-If your change is in Drupal, skip to the next section.
-
-## Merge content change in vagov-content
```

Comment:
Make sure to read the Deploy guide carefully with this process in mind. I broke this down into "partial deploy" and "full deploy" instead of tying it directly to "content changes deploy."